src/util-frontend.js

```diff
@@ -213,3 +213,96 @@ export function getToastErrorTimeout() {
 
     return errorTimeout;
 }
+
+class RelativeTimeFormatter {
+    /**
+     * Default locale and options for Relative Time Formatter
+     */
+    constructor() {
+        this.locale = currentLocale();
+        this.options = { numeric: "auto" };
+        this.rtf = new Intl.RelativeTimeFormat(this.locale, this.options);
+    }
+
+    /**
+     * Method to get the singleton instance
+     * @returns {object} Intl.RelativeTimeFormat instance
+     */
+    getInstance() {
```

Comment:
Let's inline this. As it is being used, you can just use the field as is.

Also I don't think this is a singleton

Reply:
Yes, makes sense. Have removed this function, instance can be accessed directly.